.github/prompts/tspconfig_add_missing_languages.prompt.md

```diff
@@ -0,0 +1,14 @@
+---
+mode: agent
+model: Claude Sonnet 4
+---
+
+Add a language that is missing (for instance, adding Go, Python, Javascript C# or Rust support), using these rules:
+
+- New emitter configurations should be added as a child of the "options:" section, after any other emitters and their options.
```

Comment:
should it start saying where the emitter configurations need to go? tspconfig.yaml? or it is not needed

Reply:
It does a good job of seeing the similarity between the two files, so I think we're okay.